app/resources/v1/announcement_resource.rb

```diff
@@ -24,16 +24,15 @@ def self.apply_filter(records, filter, value, options)
 
 private
   def self.get_previous_bulletin(bulletin_id)
-    bulletin = Basechurch::Bulletin.where(id: bulletin_id).first
-    Basechurch::Bulletin.latest.
-                         where('published_at < ?', bulletin.published_at).
-                         where(group_id: bulletin.group_id).
-                         first
+    bulletin = Bulletin.where(id: bulletin_id).first
+    Bulletin.latest.
+             where('published_at < ?', bulletin.published_at).
```

Comment:
Use 2 (not 9) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.